.github/workflows/sourcegraph.yml

```diff
@@ -3,24 +3,25 @@ on:
   push:
     branches:
       - main
-      - olafurpg/lets-go
-    pull_request:
+
+  pull_request:
+
 jobs:
   scip:
     runs-on: ubuntu-latest
     name: "Upload SCIP"
     steps:
       - uses: actions/checkout@v2
-      - uses: olafurpg/setup-scala@v13
+      - uses: actions/setup-java@v3
       - uses: actions/setup-go@v2
         with:
           go-version: "1.15.6"
```

Comment:
Is go actually needed? I know in my uploads I don't include this and it seems to work fine?

Reply:
This might be a remnant from the time when we build `src` from source. If CI is green without this step then it's fine to remove it

Reply:
👍🏼 I'll send in another pr after this one and double-check. I don't want to edit this one again because I opened it before my first pr was merged, so you'll have to keep clicking run 😆 . If I open another I should be able to see the result and change if necessary.